### DISCUSSION TITLE
Validation Erros

### DISCUSSION THREAD

**Discussion by danik121** [Megabit/Blazorise#5782] [Q&A]
There are possibilities to read out all error messages from validations?
something like that Validations.Errors or Validations.Messages ?

**Comment by stsrki** [comment]
You should be able to use `StatusChanged` parameter to get all the messages.

```
<Validations StatusChanged="@OnStatusChanged" />
```

```
Task OnStatusChanged(ValidationsStatusChangedEventArgs eventArgs)
{
  // do the magic
}
```

**Reply by danik121** [reply]
if i use 
```
        <Feedback>
            <ValidationError>Enter valid email!</ValidationError>
        </Feedback>

```
then eventArgs.messages has only one message "One or more fields have an error. Please check and try again"
